server_addon/applications/client/ayon_applications/addon.py

```diff
@@ -134,6 +154,17 @@ def cli(self, addon_click_group):
                 default=None
             )
         )
+        (
+            main_group.command(
+                self._cli_launch_applications,
+                name="launch",
+                help="Launch application"
+            )
+            .option("--app", help="Application name")
+            .option("--project", help="Project name")
+            .option("--folder", help="Folder path")
+            .option("--task", help="Task name")
```

Comment:
Should we also add `required=True`?

Reply:
Isn't that by default when `default` is not defined? If not then yes.

Reply:
No it was not, added `required=True`.